modules/Kconfig.mcux

```diff
@@ -1,12 +1,13 @@
 # MCUXpresso SDK
 
-# Copyright (c) 2016, Freescale Semiconductor, Inc.
+# Copyright 2016,2024 NXP
```

Comment:
@yvanderv provide guidance

Reply:
I will retain the Freescale Copyright and add NXP, this is after a conversation with @yvanderv